plugins/woocommerce-blocks/tests/e2e/playwright-utils/test.ts

```diff
@@ -161,12 +161,9 @@ const test = base.extend<
 			window.localStorage.clear();
 		} );
 
-		const cliOutput = await cli(
+		await cli(
 			`npm run wp-env run tests-cli wp db import ${ DB_EXPORT_FILE }`
 		);
-		if ( ! cliOutput.stdout.includes( 'Success: Imported ' ) ) {
-			throw new Error( `Failed to import ${ DB_EXPORT_FILE }` );
-		}
 	},
```

Comment:
Nice catch! That check was needed because the `cli` utility always resolved the promise, regardless of whether the executed script threw an error. I made it reject on an error in https://github.com/woocommerce/woocommerce/pull/46409/commits/3a60fac18769ef218a0157ebf451f363ff6db73c, so now we don't need these explicit checks as it's handled inside the utility.